tests/core/test_tracker_stores.py

```diff
@@ -135,13 +135,27 @@ def __init__(self, domain, url, port, db, password, record_exp, event_broker=Non
         )
 
 
-def test_tracker_store_from_string(default_domain: Domain):
+class HostExampleTrackerStore(RedisTrackerStore):
+    pass
+
+
+def test_tracker_store_deprecated_url_argument_from_string(default_domain: Domain):
     endpoints_path = "data/test_endpoints/custom_tracker_endpoints.yml"
     store_config = read_endpoint_config(endpoints_path, "tracker_store")
 
+    with pytest.warns(DeprecationWarning):
+        tracker_store = TrackerStore.find_tracker_store(default_domain, store_config)
+
+    assert isinstance(tracker_store, URLExampleTrackerStore)
+
+
+def test_tracker_store_with_host_argument_from_string(default_domain: Domain):
+    endpoints_path = "data/test_endpoints/custom_tracker_test_endpoints.yml"
+    store_config = read_endpoint_config(endpoints_path, "tracker_store")
+
     tracker_store = TrackerStore.find_tracker_store(default_domain, store_config)
```

Comment:
should we also test there is no warning?

Reply:
tbh forgot you could do this 😅, but of course